core/src/Streamly/Internal/Data/Parser/ParserD.hs

```diff
@@ -1932,8 +1933,14 @@ sequence (Fold fstep finitial fextract) (D.Stream sstep sstate) =
             IPartial ps1 -> do
                 pres <- pstep ps1 a
                 case pres of
-                    Partial n ps2 -> return $ Partial n (Just' $ Parser pstep (return $ IPartial ps2) pextr, ss, fs)
-                    Continue n ps2 -> return $ Continue n (Just' $ Parser pstep (return $ IPartial ps2) pextr, ss, fs)
+                    Partial n ps2 ->
+                        let newP = Just'
+                                $ Parser pstep (return $ IPartial ps2) pextr
```

Comment:
You can move `Just'` to the next line (Just' newP) so that you do not have to break this line.